### PR TITLE
types: use typings provided by remark

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Starting with [remark version 11](https://github.com/remarkjs/remark/releases/tag/11.0.0), remark has provided typings as part of the package.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -158,7 +158,7 @@ async function runRemark(document: vscode.TextDocument, range: vscode.Range): Pr
 		if (result.messages.length !== 0) {
 			let message = '';
 
-			result.messages.forEach((value: string) => {
+			result.messages.forEach((value) => {
```
Comment (ChristianMurphy) [inline src/extension.ts:161]: in this case, `value` is a [`VFileMessage`](https://github.com/vfile/vfile-message/blob/04d001493de202ffad70110bb194739a91f312d3/types/index.d.ts#L9-L93)

**Review by asbjornu — COMMENTED** [review]
This looks good, but if you have ideas for how to exercise the touched code in a test so we know for a fact that it works, that would be awesome!

**Comment by ChristianMurphy** [comment]
> but if you have ideas for how to exercise the touched code in a test so we know for a fact that it works

This corrects the typing, which is tested by `tsc`.
The implementation is the same as before.

Additional unit tests are wonderful, but would likely be out of scope of a typing fix. (?)

**Comment by asbjornu** [comment]
> This corrects the typing, which is tested by `tsc`.

Ok, fair enough.

**Comment by asbjornu** [comment]
Thanks, @ChristianMurphy! 🙏

**Pull request merged**